TESTPLAN.md

```diff
@@ -0,0 +1,47 @@
+Manual testing of the GCloud Plugin for Eclipse.
+
+# Install the plugin
+
+1. Launch Eclipse 4.5.
+
+2. If necessary, remove the old version of the plugin.
+  1. Help > Installation details
+  2. Click "Google Cloud Platform for Eclipse 4.5". (If this is not present,
+    move on to step 3 below.)
+  3. Click the Uninstall... button.
+  4. Click the Finish button.
+  5. Click the "Yes" button in the dialog that prompts you to restart Eclipse.
+2. ????
+3. Restart Eclipse
+4. Check installation details. 
+
+# Create a project
```

Comment:
These are related to single project situations.  We'll need something similar for EAR-style multi-service projects.